src/util/clientLoader.js

```diff
@@ -11,13 +11,23 @@ export default () => {
 		],
 		intents: [
 			Intents.FLAGS.GUILDS,
+			Intents.FLAGS.GUILD_PRESENCES,
```

Comment:
Cet intent est-t-il vraiment nécessaire ? Il servirait à recevoir un event concernant le changement de présence (statut, activité) des membres : https://discord.com/developers/docs/topics/gateway#list-of-intents

Reply:
Si n'est pas ajouté le bot ne démarre pas, j'ai mis 1 jour à trouver une solution, dès qu'il a été ajouté tout fonctionnait